platform/platform-paxweb-jettyconfig/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,43 +15,24 @@
 
     <reference id="securityLogger" interface="ddf.security.audit.SecurityLogger" availability="optional" />
 
+    <!-- Global ServletFilters -->
+    <service id="clientInfoFilter" interface="org.codice.ddf.platform.filter.HttpFilter">
+        <bean class="org.codice.ddf.pax.web.jetty.ClientInfoFilter"/>
+    </service>
+
     <bean id="responseFilter" class="org.codice.ddf.pax.web.jetty.ResponseFilter">
         <cm:managed-properties
-                persistent-id="org.codice.ddf.platform.response.filter.ResponseHeaderConfig"
-                update-strategy="container-managed"/>
-    </bean>
-
-    <bean id="executor" class="java.util.concurrent.Executors" factory-method="newSingleThreadScheduledExecutor">
-        <argument ref="securityJavaSubjectThreadFactory"/>
+          persistent-id="org.codice.ddf.platform.response.filter.ResponseHeaderConfig"
+          update-strategy="container-managed"/>
     </bean>
+    <service ref="responseFilter" interface="org.codice.ddf.platform.filter.HttpFilter"/>
 
-    <reference-list id="filterList" availability="optional" interface="javax.servlet.Filter" />
-
-    <bean id="delegateServletFilter"
-          class="org.codice.ddf.pax.web.jetty.DelegateServletFilter">
-        <argument ref="filterList"/>
-    </bean>
-
-    <bean id="filterInjector" class="org.codice.ddf.pax.web.jetty.FilterInjector" init-method="init"
-          destroy-method="destroy">
-        <argument type="java.util.List">
-            <list value-type="javax.servlet.Filter">
-                <bean class="org.codice.ddf.pax.web.jetty.ClientInfoFilter"/>
-                <ref component-id="responseFilter"/>
-                <bean class="org.codice.ddf.pax.web.jetty.SecurityJavaSubjectFilter"/>
-                <ref component-id="delegateServletFilter"/>
-                <bean class="org.codice.ddf.pax.web.jetty.DoPrivilegedFilter"/>
-            </list>
-        </argument>
-        <argument ref="executor"/>
-    </bean>
-
-    <bean id="securityJavaSubjectThreadFactory"
-          class="org.codice.ddf.platform.util.StandardThreadFactoryBuilder"
-          factory-method="newThreadFactory">
-        <argument value="securityJavaSubjectThread"/>
-    </bean>
+    <service id="securityJavaSubjectFilter" interface="org.codice.ddf.platform.filter.HttpFilter">
```

Comment:
❓ Did we lose the ability to make sure these core handlers are executed in the proper order?  I guess we could use service ranking if we need it again.

Reply:
Hm... I'm not sure what guarantees we have about execution order anymore. If we need to preserve the existing order (ClientInfoFilter, ResponseFilter, SecurityJavaSubjectFilter, ServletMetrics, DoPrivilegedFilter), I'll look into it

Reply:
Originally I was intentional abut making sure the filter delegate went after the security java subject filter so that the delegated filters could assume a subject would be available if the user was logged in.  I don't think the current filters care about that currently.  It is probably ok how you have it currently.

Reply:
Though I wonder if it will impact the Java security manager policy.  Before we were able to keep policies details from leaking up to global filters.

Reply:
Good point. I'll see what I can do to make the DoPrivilegedFilter run last. Not sure if the ServiceTracker supports service ranking as I'm using it, but that would be easiest

Reply:
You could do something similar to our `SortedServiceList`.

https://github.com/codice/ddf/blob/master/platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/SortedServiceList.java#L57-L64